modules/ROOT/pages/using-maven-with-mule.adoc

```diff
@@ -56,6 +56,20 @@ Example XML:
 <packaging>mule-application</packaging>
 ----
 
+== Configure Your Maven Installation
+
+Even though Anypoint Studio automatically configures Maven to work with Mule when you create a new project, some scenarios require you to manually configure your Maven installation to work with Mule. For example when following continuous integration or continuous deployment (CI/CD) practices.
```

Comment:
The text "For example.." is not a sentence. I suggest: 

"For example, you must manually configure your Maven installation when following continuous integration or continuous deployment (CI/CD) practices.

or adding to the prior sentence with a semicolon before the text